hyperv-tools/Convert-WindowsImage/Convert-WindowsImage.psm1

```diff
@@ -1584,7 +1584,7 @@ You can use the fields below to configure the VHD or VHDX that you want to creat
                     $VhdPath      = Join-Path  -Path $VhdPath -ChildPath $vhdNameTemp
                 }
 
-                Write-Debug -Message "  Temporary $VhdFormat path is: `"$VhdPath`""
+                Write-Debug -Message "  Temporary $VhdFormat path is: `�$VhdPath`�"
```

Comment:
Heaps of corrupts on double quotes - will need to fix these before the PR would go through.

Reply:
Thanks for the find. Must've pasted in RTF or something on the PR. I've updated with a new commit.